metrics/network/remote_network/remote-networking-test-common.sh

```diff
@@ -59,6 +59,17 @@ function get_runtime {
 	fi
 }
 
+# This function will check the environment in host B
```

Comment:
Do not use future tense when commenting code. Stick to present tense:
This function checks the environment in host B.